extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/DevServicesConfig.java

```diff
@@ -97,57 +101,56 @@ public class DevServicesConfig {
     public boolean showLogs;
 
     /**
-     * Keycloak start command.
+     * Optional. Keycloak start command.
      * Use this property to experiment with Keycloak start options, see {@link https://www.keycloak.org/server/all-config}.
-     * Note it will be ignored when loading legacy Keycloak WildFly images.
+     * Note, it is ignored when loading legacy Keycloak WildFly images.
      */
     @ConfigItem
     public Optional<String> startCommand;
 
     /**
-     * The Keycloak realm name.
-     * This property will be used to create the realm if the realm file pointed to by the `realm-path` property does not exist,
-     * default value is `quarkus` in this case.
-     * If the realm file pointed to by the `realm-path` property exists then it is still recommended to set this property
-     * for Dev Services for Keycloak to avoid parsing the realm file to determine the realm name.
+     * Optional. The name of the Keycloak realm. The default value is `quarkus`.
+     *
+     * This property is used to create the realm when the realm file specified by the `realm-path` is not found.
      *
+     * It's recommended to always set this property so that Dev Services for Keycloak can identify the realm name without
+     * parsing the realm file.
      */
     @ConfigItem
     public Optional<String> realmName;
 
     /**
-     * Indicates if the Keycloak realm has to be created when the realm file pointed to by the `realm-path` property does not
-     * exist.
+     * Specifies whether to create the Keycloak realm when no realm file is found at the `realm-path`.
      *
-     * Disable it if you'd like to create a realm using Keycloak Administration Console
-     * or Keycloak Admin API from {@linkplain io.quarkus.test.common.QuarkusTestResourceLifecycleManager}.
+     * Set to `false` if the realm is to be created using either the Keycloak Administration Console or
+     * the Keycloak Admin API provided by {@linkplain io.quarkus.test.common.QuarkusTestResourceLifecycleManager}.
      */
     @ConfigItem(defaultValue = "true")
     public boolean createRealm;
 
     /**
-     * The Keycloak users map containing the username and password pairs.
-     * If this map is empty then two users, 'alice' and 'bob' with the passwords matching their names will be created.
-     * This property will be used to create the Keycloak users if the realm file pointed to by the `realm-path` property does
-     * not exist.
+     * A map of Keycloak usernames to passwords.
+     *
+     * If empty, default users `alice` and `bob` are created with their names as passwords.
```

Comment:
@rolfedh I like this style more, see `Map<String, String>` is effectively optional, i.e, it can be empty and users are not required to configure it

Reply:
Updated. Thanks.